pkg/addons/velero/velero.go

```diff
@@ -51,6 +54,12 @@ var helmValues = map[string]interface{}{
 			},
 		},
 	},
+	"podLabels": map[string]interface{}{
+		"component": "velero",
+	},
```

Comment:
KOTS uses this in a label selector when checking if the Velero/NodeAgent pods have restarted post-configuration.